python/docs/blog/posts/2023-11-06-four-pillars-of-realtime-ai.qmd

```diff
@@ -0,0 +1,115 @@
+---
+date: 2023-Nov-06
+author: Ben Chambers
+categories:
+ - real-time AI
+ - GenAI
+title: "Always On: Four Pillars of Real-time Generative AI"
+---
+
+With the rise of Large-Language Models (LLMs), it is now possible for everyone to create AI-powered applications and agents.
+Starting with foundation models, developers can rapidly prototype the application and then jump into advanced techniques by tuning the prompt or introducing Retrieval-Augmented Generation (RAG), Few-Shot Learning, and Fine-Tuning, all without having to dive deep into machine learning and model learning.
+While current LLM uses often revolve around a few simple applications – chatbots, content summarization, etc. – I see a vast potential in real-time applications.
+
+Imagine a world where LLMs are constantly watching the world around and looking out for your interests.
+Instead of using GPT to draft an e-mail, ask the agent to send it for you.
+As e-mails arrive, they are constantly triaged, with some receiving immediate (generated) responses, most being provided in occassionally summaries for your attention, and only the most important actually interrupting you.
+As news articles are posted, they may be filtered and grouped based on your interests and summarized when you request.
+[BeepGPT](https://epinzur.github.io/posts/001-openai-fine-tuning-1/post.html), an example agent that automatically listens to messages on Slack and pings you if appropriate is just the beginning.
+Such agents are "always on" and tapped into the firehose of evetns happening around you.
```

Comment:
>  Imagine a world where LLMs are constantly watching the world around you* and looking out for your 
interests. 

perhaps adds "you" 

Alternate paragraph if you like any of this; it just reverses the flow to start with the problem first. 

"""
Imagine a world where LLMs are constantly watching the world around you and looking out for your 
interests. 

Instead of being interrupted by emails or chats and having to read, filter, and draft responses [for important ones], agents will constantly triage and take the appropriate action for each. Some will receive immediate (generated) responses, some will be summarized for later review, then only the time-sensitive [or important] ones will actually [notify you/interrupt you]. For example, [BeepGPT](https://epinzur.github.io/posts/001-openai-fine-tuning-1/post.html), is an agent that automatically listens to messages on Slack and pings you, if appropriate. Such agents are ...
"""

The news articles sentence feels somewhat out of place in between the email -> beepgpt connection.